puppet-agent-debian/Dockerfile

```diff
@@ -15,7 +15,7 @@ LABEL org.label-schema.vendor="Puppet" \
       com.puppet.dockerfile="/Dockerfile"
 
 RUN apt-get update && \
-    apt-get install -y wget=1.16-1 && \
+    apt-get install -y wget && \
```

Comment:
@garethr I noticed this too. Was there a reason you were installing a specific version of wget here?